runtime/dancebox/src/lib.rs

```diff
@@ -920,6 +920,17 @@ construct_runtime!(
     }
 );
 
+#[cfg(feature = "runtime-benchmarks")]
+mod benches {
+    frame_benchmarking::define_benchmarks!(
+        [pallet_author_noting, AuthorNoting]
+        [pallet_configuration, Configuration]
+        [pallet_registrar, Registrar]
+        [pallet_invulnerables, Invulnerables]
+        [pallet_xcm_benchmarks::generic, pallet_xcm_benchmarks::generic::Pallet::<Runtime>]
```

Comment:
Unless it's somewhere else, this list is missing pallet_pooled_staking and frame_system:

```
            list_benchmark!(list, extra, frame_system, SystemBench::<Runtime>);

            list_benchmark!(
                list,
                extra,
                pallet_pooled_staking,
                PalledPooledStaking::<Runtime>
            );
```

Reply:
Yes, I might have missed the staking one after merging master. I'll add them.